docs/contributor/performance-tests/metrics/metrics.md

```diff
@@ -0,0 +1,690 @@
+
+Kubernetes API request duration reach for `GET` operations at peak `~750ms` and for `LIST` operations `~900ms`.
+
+### Multiple pods and services scraped from multiple receivers
+
+Setup:
+
+Avalanche load generator configuration:
+- Metric Count: 500
+- Metric Series: 20
+- Number of label: 10
+
+Metric agent:
+
+Metric agent pipeline configuration `metrics/prometheus` configured with an additional receiver `prometheus/app-services` as below.
+```yaml
+apiVersion: v1
+data:
+  relay.conf: |
+    extensions:
+        health_check:
+            endpoint: ${MY_POD_IP}:13133
+    service:
+        pipelines:
+            metrics/prometheus:
+                receivers:
+                    - prometheus/app-pods
+                    - prometheus/app-services
+                processors:
+                    - memory_limiter
+                    - resource/delete-service-name
+                    - resource/insert-input-source-prometheus
+                    - batch
+                exporters:
+                    - otlp
+```
+Following service created for Avalanche load generator to enable metric scrape from second receiver `prometheus/app-services`
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "8080"
+    prometheus.io/scrape: "true"
+  labels:
+    app: avalanche-metric-load-generator
+  name: avalanche-metric-load-generator
+spec:
+  clusterIP: None
+  ports:
+  - name: http-metrics
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: avalanche-metric-load-generator
+  sessionAffinity: None
+  type: ClusterIP
+```
+
+Purpose of this test was to determine scrape behavior of metric agent and especially memory impact with multiple receiver under high load.
+
+Test metric generator started with 10 instances and metrics are scraped by the `prometheus/app-pods` receiver and `prometheus/app-services` receiver in same time independently.
+Metric generator instances were increased with `2` instance every minute to simulate a ramp-up of load.
+
+With 14 instances, metric agent reached the stable peak of `280K` metric samples per scrape loop in total, which is  similar to the single receiver performance.
+Additional load on top of this setup caused refusal of incoming metrics by the `memory_limiter` processor.
+
+Following graph shows accepted metrics during test by the both receivers
+![Metric agent accepted metrics](./assets/agent-multi-receiver-accepted.jpg)
+
+Following graph shows refused metrics during test from processor
+![Metric agent accepted metrics](./assets/agent-multi-receiver-refused.jpg)
+
+CPU utilization of the metric agent
+![Metric agent accepted metrics](./assets/agent-multi-receiver-cpu.jpg)
+
+Memory utilization of the metric agent
+![Metric agent accepted metrics](./assets/agent-multi-receiver-memory.jpg)
+
+### Scrape sample limit test
+
+Setup:
+
+Avalanche load generator configuration to generate `2000` metric points per scrape :
```

Comment:
```suggestion
Avalanche load generator was configured to generate `2000` metric points per scrape:
```
Same here, why are some elements formatted in code font? Does any of our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-code-font-for-these-items) apply?